content/articles/terraform-provider.markdown

```diff
@@ -0,0 +1,120 @@
+---
+title: Terraform Provider
+excerpt: How to use DNSimple's Terraform provider to manage your DNS
+categories:
+- Services
```

Comment:
All of the other articles in the Services category seem to relate to our one-click services. I'm not sure this one fits. 😄 

```suggestion
```